pkg/commands/entrypoints/entrypoints.go

```diff
@@ -165,13 +176,7 @@ func (c *EntrypointsCommand) Run(ctx context.Context, args []string) error {
 		return fmt.Errorf("failed to parse flags: %w", err)
 	}
 
-	parsedArgs := f.Args()
```

Comment:
I think we still want this, only it should validate there are 0 args otherwise, show help since we dont expect any args.